docs/loading.html

```diff
@@ -1,11 +1,9 @@
 <!doctype html>
 <html lang="en">
 <title>Loading!</title>
+<link rel="stylesheet" href="css/site.css">
```

Comment:
Is there any way this can be avoided? We want this loading.html page to load pretty much instantly so I'd like to avoid loading a bunch of CSS if it's not needed. Can we just inline all of the styles this one needs in the below style tag? Is it much extra than what is already there?

Reply:
I just realized this is likely for the icon. If that's correct can we link to css/icons.css instead?

Reply:
Gotcha. I added this for the embeded font and icon. But I see your point. I just pushed a new version more simplified.